deepmd/dpmodel/descriptor/hybrid.py

```
@@ -332,7 +333,9 @@ def call(
                 # mixed_types is True, but descrpt.mixed_types is False
                 assert nl_distinguish_types is not None
                 nl = nl_distinguish_types[:, :, nci]
-            odescriptor, gr, g2, h2, sw = descrpt(coord_ext, atype_ext, nl, mapping)
+            odescriptor, gr, g2, h2, sw = descrpt(
```

Ruff (0.15.12)

[warning] Unpacked variable `g2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)